react/features/video-menu/components/native/RemoteVideoMenu.js

```diff
@@ -172,8 +198,20 @@ class RemoteVideoMenu extends PureComponent<Props> {
 function _mapStateToProps(state, ownProps) {
     const kickOutEnabled = getFeatureFlag(state, KICK_OUT_ENABLED, true);
     const { participant } = ownProps;
+    const participantIDS = [];
+
     const { remoteVideoMenu = {}, disableRemoteMute } = state['features/base/config'];
     let { disableKick } = remoteVideoMenu;
+    const localParticipant = getLocalParticipant(state);
+    const remoteParticipants = getRemoteParticipants(state);
+
+    localParticipant && participantIDS.push(localParticipant?.id);
+
+    remoteParticipants.forEach(p => {
+        participantIDS.push(p?.id);
+    });
+
+    const isParticipantIDAvailable = participantIDS.find(partID => partID === participant.id);
```

Comment:
IIRC there is a getParticipantById helper.

Reply:
I don't see a use for it here because I already have the participant object and the check needs the participant id.

Reply:
I don't see any use for that helper function. I already have the participant, and I only need its id to check if it's the same with any element from the participantIDS array.

Reply:
Well, you are building an array of ids and then searching inside it. Assuming the participant was the last, it will take you a full pass to find it. If you use the helper it will be a single check since the participants are stored in a map. So it'll be O(n) vs O(1).

Not that this is a hot path, but the code reads weird and I fear it may lead to confusion in the future.

Reply:
You're right. Thanks and this is updated!